charts/openebs/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
-version: 2.12.0
+version: 3.0.0
 name: openebs
-appVersion: 2.12.0
+appVersion: 3.0.0
```

Comment:
This also can change to 2.12.1. We will be making change to maya to remove the class generation from m-apiserver.